UI/Login.tsx

```diff
@@ -16,7 +16,7 @@ import {Button2} from './components/Button2';
 
 export const Login = ({navigation}: NavigationProps) => {
   function onLogin() {
-    Alert.alert('Login successful!');
+    navigation.navigate('Search', {});
```

Comment:
Can you delete navigation.navigate('Search', {}); ?  this could throw an error because we don't need to explicitly tell the app to navigate to main screens once the user has been authenticated; react-native-navigation will do it for us. We just need to implement the logic for it, I opened an authentication branch, so I'm working on hooking the endpoints.